package.json

```diff
@@ -86,8 +86,8 @@
     "jest-coverage": "yarn jest --coverage",
     "lint": "eslint . --cache --report-unused-disable-directives --ext js,jsx,ts,tsx,md",
     "lint-es5-build": "eslint --no-eslintrc --no-ignore --env=browser packages/*/build-es5",
-    "lint:md": "yarn --silent lint:md:ci --fix",
-    "lint:md:ci": "prettylint '**/*.{md,yml,yaml}' --ignore-path .gitignore",
+    "lint:prettier": "yarn --silent lint:prettier:ci --fix",
```

Comment:
Thought I'd also rename the script while on it since it's no longer just `md`. Simen mentioned this name being misleading some time ago.